.bazelrc

```diff
@@ -61,12 +61,14 @@ build:oss-fuzz --@rules_fuzzing//fuzzing:cc_engine_sanitizer=none
 build:jazzer --//fuzzing:java_engine=//fuzzing/engines:jazzer
 build:jazzer --@rules_fuzzing//fuzzing:cc_engine_instrumentation=jazzer
 build:jazzer --@rules_fuzzing//fuzzing:cc_engine_sanitizer=none
-# Workaround for https://github.com/bazelbuild/bazel/issues/11128
-build:jazzer --//fuzzing:cc_engine_sanitizer=none
 
 # Jazzer + ASAN
 build:asan-jazzer --//fuzzing:java_engine=//fuzzing/engines:jazzer
```

Comment:
BTW, note that this is not copy-pasteable, either: `//fuzzing/engines:jazzer` must also be prefixed with `@rules_fuzzing` in user repos.